src/navigator/types.tsx

```diff
@@ -347,8 +348,7 @@ export type QRTabParamList = {
   [Screens.QRScanner]:
     | {
         scanIsForSecureSend?: true
```

Comment:
this prop is still needed for *visual* differences with QR scanner for the secure send flow

Reply:
could be nice to rename it like `showSecureSendStyling` (or something) to make that more explicit